test/neo/client/request/internal/GetAll.d

```diff
@@ -433,6 +435,8 @@ private struct Controller
         controller.handle(this.conn, this.context,
             this.request_event_dispatcher, this.fiber);
 
+        this.conn.flush();
```

Comment:
Actually, this flush should be added here https://github.com/sociomantic-tsunami/swarm/blob/v4.x.x/src/swarm/neo/client/mixins/SuspendableRequestCore.d#L814